tests/conftest.py

```diff
@@ -85,6 +86,12 @@ def pytest_addoption(parser):
         default=DEFAULT_SKIP_SERVER_SETUP,
         help="Skip server setup operations",
     )
+    parser.addoption(
+        "--to-review-enabled",
+        action="store_true",
+        default=DEFAULT_TO_REVIEW_ENABLED,
+        help="Include tests and related to the to-review config option",
+    )
```

Comment:
I think having this as a CLI flag (or somehow not relying on introspection alone) might be useful. For example, we expect `to-review` to be disabled on the dev server. Say it was somehow enabled. If we used introspection to determine which signer test to run, it would pick the "right" one, but it would break our "assertion" that `to-review` was disabled.
What are your thoughts there?

Reply:
I agree, I hadn't looked at it this way.

If we want to check that the server has a particular setting, then we have to store the expected value somewhere.
Either, we introspect and do ugly stuff like `"DEV" in server_info["name"] and to_review is False` or we pass it in CLI like you did.

If we want to be smarter here (with rules, or default values by ENV etc.) then we can still improve it over time.

tldr; ignore my comment and leave the CLI flag.